var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -50,9 +50,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
     version("3.1.00", sha256="b935c9b780e7330bcb80809992caa2b66fd387e3a1c261c955d622dae857d878")
     version("3.0.00", sha256="c00613d0194a4fbd0726719bbed8b0404ed06275f310189b3493f5739042a92b")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("cxx", type="build")  # Kokkos requires a C++ compiler
```

Comment:
```suggestion
    depends_on("cxx", type="build")
```

Reply:
While we don't need the comment, removing it would not justify re-starting the CI pipeline.